packages/rum-core/src/state.js

```diff
@@ -24,7 +24,11 @@
  */
 
 const __DEV__ = process.env.NODE_ENV !== 'production'
+const state = {
+  //  Time when agent code is executed and patching of modules happens
+  patchedTime: null,
+  // Time when the document is last backgrounded
+  lastHiddenStart: Number.MIN_SAFE_INTEGER
```

Comment:
@jahtalab I added this check as we don't want to compare undefined with number values. Just an extra guard.